docs/developing-locally.rst

```diff
@@ -9,24 +9,53 @@ Setting Up Development Environment
 
 Make sure to have the following on your host:
 
-* virtualenv_;
-* pip;
-* PostgreSQL.
+* Python 3.6
+* PostgreSQL_.
+* Redis_, if using Celery
 
 First things first.
 
-#. `Create a virtualenv`_.
+#. Create a virtualenv: ::
 
-#. Activate the virtualenv you have just created.
+    $ python3 -m venv <virtual env path>
+
+#. Activate the virtualenv you have just created: ::
+
+    $ source <virtual env path>/bin/activate
 
 #. Install development requirements: ::
 
     $ pip install -r requirements/local.txt
 
-#. Create a new PostgreSQL database (note: if this is the first time a database is created on your machine you might need to alter a localhost-related entry in your ``pg_hba.conf`` so as to utilize ``trust`` policy): ::
+#. Create a new PostgreSQL database using createdb_: ::
 
     $ createdb <what you've entered as the project_slug at setup stage>
 
+   .. note::
+
+       if this is the first time a database is created on your machine you might need to alter
+       a localhost-related entry in your ``pg_hba.conf`` so as to utilize ``trust`` policy, see the
+       `postgres documentation`_ for more details.
+
+#. Set the environment variables for your database(s): ::
+
+    $ export DATABASE_URL=postgres://<project_slug>
```

Comment:
if use the way I configure the postgres, then must also pass the user and password as a parameter, I'm also explicit about the host and port

```
export DATABASE_URL=postgres://postgres:password@127.0.0.1:5432
```

Reply:
Agreed. I think it would be better to include the entire thing here, actually:

```
export DATABASE_URL=postgres://<user>:<password>@127.0.0.1:5432/<db_name>
```